src/rtk_defines.h

```diff
@@ -183,7 +183,7 @@
 typedef uint32_t pthread_t;
 typedef uint32_t pthread_mutex_t;
 typedef int lock_t;
-typedef int DIR;
+// typedef int DIR;
```

Comment:
Not sure the importance of DIR, but why would bringing back in the EVB source would need this removed?